### PR TITLE
Flavors2

### PR DESCRIPTION
**Pull request by fepitre**
From the work and comments you have made, xinit script is in 'gui-agent-linux' and Thunar custom actions and options are in 'core-agent-linux' as it is the case for Nautilus.

### CODE REVIEW

**Comment by fepitre** [comment]
When everything will be ok, I will also prepare a PR in the builder for adding XFCE flavor in the setup and the example config files.

File: template_scripts/04_install_qubes.sh [hunk 1]
```diff
@@ -23,12 +23,13 @@ if [ -n "$USE_QUBES_REPO_VERSION" ]; then
 fi
 
 echo "--> Installing RPMs..."
-if [ "$TEMPLATE_FLAVOR" != "minimal" ]; then
-    installPackages packages_qubes.list || RETCODE=1
+if [ "x$TEMPLATE_FLAVOR" != "x" ]; then
+	installPackages packages_qubes_${TEMPLATE_FLAVOR}.list || RETCODE=1
```
Comment (marmarek) [inline template_scripts/04_install_qubes.sh]: Please use 4 spaces indentation.

File: template_scripts/extras/50-xfce-desktop.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+[ -d "$XDG_CACHE_HOME" ] || mkdir "$XDG_CACHE_HOME"
+
+if which xdg-user-dirs-update >/dev/null 2>&1; then
+    xdg-user-dirs-update
+fi
+
+QT_QPA_PLATFORMTHEME="qt5ct"
+export QT_QPA_PLATFORMTHEME
+
+xfsettingsd &
```
Comment (marmarek) [inline template_scripts/extras/50-xfce-desktop.sh]: This file is not needed here anymore (it's in qubes-gui-agent-xfce package).

**Comment by marmarek** [comment]
Yes, please.

**Comment by fepitre** [comment]
I squashed almost all the commits. Kept the first one for possible further improvement of the selection.

**Pull request merged**